<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Form1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>فرم اتصال</string>
    </property>
    <widget class="QLineEdit" name="IP_inForm1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ECF0F1;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
</string>
     </property>
     <property name="placeholderText">
      <string>127.0.0.2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #F1C40F;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
</string>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Port_inForm1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ECF0F1;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
</string>
     </property>
     <property name="placeholderText">
      <string>12345</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #F1C40F;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 2px 2px;
border-radius: 5px;
</string>
     </property>
     <property name="text">
      <string>port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UserName_inform1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ECF0F1;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
</string>
     </property>
     <property name="placeholderText">
      <string>User Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Passwor_inForm1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ECF0F1;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BT_form1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #F1C40F;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
}

QPushButton:hover {
    background-color: #1ABC9C;
    color: black;
}

QPushButton:pressed {
    background-color: #16A085;
    border: 1px solid #16A085;
}
</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusIcon_form1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: red;
    border-radius: 10px;
    border: 1px solid black;
    min-width: 20px;
    min-height: 20px;
    max-width: 20px;
    max-height: 20px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Form_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>فرم محاسبات</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="RB_Max">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    color: BLACK;
}
QRadioButton::indicator {
    width: 16px;
    height: 16px;
    border-radius: 8px;
    border: 2px solid #1ABC9C;    
    background-color: #2C3E50;
}

QRadioButton::indicator:checked {
    background-color: #1ABC9C; 
    border: 2px solid #16A085;
}

QRadioButton::indicator:unchecked:hover {
    border: 2px solid #F39C12;
}
</string>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RB_min">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    color: BLACK;
}
QRadioButton::indicator {
    width: 16px;
    height: 16px;
    border-radius: 8px;
    border: 2px solid #1ABC9C;    
    background-color: #2C3E50;
}

QRadioButton::indicator:checked {
    background-color: #1ABC9C; 
    border: 2px solid #16A085;
}

QRadioButton::indicator:unchecked:hover {
    border: 2px solid #F39C12;
}
</string>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RB_normal">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    color: BLACK;
}
QRadioButton::indicator {
    width: 16px;
    height: 16px;
    border-radius: 8px;
    border: 2px solid #1ABC9C;    
    background-color: #2C3E50;
}

QRadioButton::indicator:checked {
    background-color: #1ABC9C; 
    border: 2px solid #16A085;
}

QRadioButton::indicator:unchecked:hover {
    border: 2px solid #F39C12;
}
</string>
      </property>
      <property name="text">
       <string>Normal</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="Rang_form2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ECF0F1;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Range</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BT_form2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #F1C40F;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
}

QPushButton:hover {
    background-color: #1ABC9C;
    color: black;
}

QPushButton:pressed {
    background-color: #16A085;
    border: 1px solid #16A085;
}
</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Form_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>فرم نمایش</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>TypeChar</string>
     </property>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    color: BLACK;
}
QRadioButton::indicator {
    width: 16px;
    height: 16px;
    border-radius: 8px;
    border: 2px solid #1ABC9C;    
    background-color: #2C3E50;
}

QRadioButton::indicator:checked {
    background-color: #1ABC9C; 
    border: 2px solid #16A085;
}

QRadioButton::indicator:unchecked:hover {
    border: 2px solid #F39C12;
}
</string>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    color: BLACK;
}
QRadioButton::indicator {
    width: 16px;
    height: 16px;
    border-radius: 8px;
    border: 2px solid #1ABC9C;    
    background-color: #2C3E50;
}

QRadioButton::indicator:checked {
    background-color: #1ABC9C; 
    border: 2px solid #16A085;
}

QRadioButton::indicator:unchecked:hover {
    border: 2px solid #F39C12;
}
</string>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="BT_form3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #F1C40F;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
}

QPushButton:hover {
    background-color: #1ABC9C;
    color: black;
}

QPushButton:pressed {
    background-color: #16A085;
    border: 1px solid #16A085;
}
</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>66</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ECF0F1;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Form_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>خط</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #F1C40F;
font-size: 16px;
font-weight: bold;
background-color: #2C3E50;
padding: 4px 8px;
border-radius: 5px;
}

QPushButton:hover {
    background-color: #1ABC9C;
    color: black;
}

QPushButton:pressed {
    background-color: #16A085;
    border: 1px solid #16A085;
}
</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="chartContainer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>871</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
